db/post_migrate/20180813113448_copy_status_stats_cleanup.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CopyStatusStatsCleanup < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
+
+  def change
+    safety_assured do
+      remove_column :statuses, :reblogs_count, :integer, default: 0, null: false
+      remove_column :statuses, :favourites_count, :integer, default: 0, null: false
```

Comment:
These will cause any existing uses of these columns to throw errors until the code is migrated: I might suggest waiting on this for a later release, so most people won't have old code running when these columns are removed.

Reply:
Ah, right-o. Didn't realize that had changed. (Or that this was in `post_migrate`.) Carry on, then!